scripts/install-filecoin-parameters.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+fetch_params() {
+  ./proofs/bin/paramfetch --all --verbose --json=./proofs/misc/parameters.json
+}
+
+generate_params() {
+  ./proofs/bin/paramcache
+}
+
+if ! fetch_params; then
```

Comment:
BLOCKING: We should always run `paramcache`, regardless of whether or not `./proofs/bin/paramfetch` produces a non-zero exit code. 

We should do this because it's possible that a rust-fil-proofs developer forgot to update the `parameters.json` file to reflect the state of the Rust code. Running `paramcache` will _always_ check/download the correct parameters - regardless of what's in the `parameters.json` file. This ensures that we won't exercise proofs code w/out the cache being hydrated ahead of time.